src/ansys/fluent/core/logging.py

```diff
@@ -0,0 +1,140 @@
+"""Module controlling PyFluent's logging functionality."""
```

Comment:
@raph-luc are you strongly advocating for this module or can the users live without it?

Reply:
I would expect that most users should be able to live without debug logging. Print to console stdout for warning and error messages is automatically initialized in `__init__.py`, so users generally shouldn't need to worry about it either.

I am not yet aware of any use case where users necessarily need the debug output to be able to do anything.

I believe debugging is mostly for us and other developers or more advanced users.